.travis.yml

```diff
@@ -5,6 +5,8 @@ php:
 - 7.0
 - 7.1
 - 7.2
+- 7.3
+- 7.4
```

Comment:
Travis CI fails in 5.6, 7.0, 7.1, 7.2 so those should be fixed.

Example output of 7.2
```
Problem 1
253    - mockery/mockery is locked to version 1.4.2 and an update of this package was not requested.
254    - mockery/mockery 1.4.2 requires php ^7.3 || ^8.0 -> your php version (7.2.15) does not satisfy that requirement.
```